core/templates/dev/head/profile/Signup.js

```diff
@@ -67,7 +75,7 @@ oppia.controller('Signup', [
         username: $scope.username
       }).success(function(data) {
         if (data.username_is_taken) {
-          $scope.warningText = 'Sorry, this username is already taken.'
+          $scope.warningText = 'I18N_SIGNUP_ERROR_USERNAME_TAKEN'
```

Comment:
Hm, interesting -- that's a good catch. In that case, how about renaming this to $scope.warningI18nCode or something like that?